src/python/pants/notes/2.1.x.rst

```diff
@@ -0,0 +1,160 @@
+2.1.x Stable Releases
+=====================
+
+This document describes releases leading up to the ``2.1.x`` ``stable`` series.
+
+See https://www.pantsbuild.org/v2.1/docs/release-notes-2-1 for an overview of the changes in this release.
+
+2.1.0.dev0 (10/30/2020)
+----------------------
```

Comment:
Our (old?) rst rendering would have complained that the dashes aren't as long as the title here... maybe the PyPI rendering for wheels doesn't care.